OneBusAway/ui/stops/OBAStopViewController.m

```diff
@@ -123,6 +125,8 @@ - (void)viewWillDisappear:(BOOL)animated {
     [[NSNotificationCenter defaultCenter] removeObserver:self name:UIApplicationWillEnterForegroundNotification object:nil];
 
     [self cancelTimers];
+    
+    [[OBAHandoff shared] stopBroadcasting];
```

Comment:
This method name should be the inverse of the beginning operation. i.e. `stopUpdating` instead of `stopBroadcasting`. Or you can change `updateWithStopID:withRegion:` to be `broadcastEtc:etc:`